### ISSUE TITLE
[abstract-interpretation] ESBMC crashes with the options --interval-analysis --interval-analysis-simplify

### ISSUE THREAD

**Issue by lucasccordeiro** [esbmc/esbmc#1054]
s$ esbmc intervals.c --interval-analysis --interval-analysis-simplify
ESBMC version 7.2.0 64-bit x86_64 linux
Target: 64-bit little-endian x86_64-unknown-linux with esbmclibc
Parsing
Converting
Generating GOTO Program
GOTO program creation time: 0.112s
Segmentation fault

````C
#include <assert.h>

int main()
{
  unsigned int a;

  if(a>=0 && a<=10)
    return 0;

  assert(a > 10);

  return 0;
}
````

**Comment by lucasccordeiro** [comment]
@rafaelsamenezes: could you please check this issue once you have some spare time? :-)

**Comment by rafaelsamenezes** [comment]
> @rafaelsamenezes: could you please check this issue once you have some spare time? :-)

This is fixed in #1133

**Comment by lucasccordeiro** [comment]
Thanks, @rafaelsamenezes. 

Can you please add this test case to your PR? :-)
